### PR TITLE
Update/improve two aspects of the merging process.

### PR DESCRIPTION
**Pull request by Zimmi48**
- Following discussion in #7042, we write down an advice to give time for last comments before merging potentially controversial PRs.

- We document a practice that is becoming standard in order to improve PR merging time: some other maintainer can self-assign if the main maintainer is not responding. This is also something that we discussed with @herbelin and @mattam82 at FLOC.

Note that this doesn't invalidate:

> A maintainer is expected to be reasonably reactive, but no specific timeframe is given for reviewing.

Maintainers should strive to be reactive nonetheless.

**Kind:** documentation

### CODE REVIEW

**Review by maximedenes — COMMENTED** [review]
I find "or has any chance of being controversial" too ambiguous.

File: dev/doc/MERGING.md
Comment (Zimmi48) [reply dev/doc/MERGING.md]: Good point.

**Comment by Zimmi48** [comment]
I'm still waiting on a better wording proposal because I've got none.

**Comment by Zimmi48** [comment]
@maximedenes Reworded.

**Comment by maximedenes** [comment]
did @herbelin want his comment to be addressed in this PR, or is he ok with this being merge as is?

**Comment by Zimmi48** [comment]
@herbelin If you want me to add something, please write the sentence because I don't know how to put this.

**Comment by ppedrot** [comment]
This should probably be merged as-is then...

**Comment by herbelin** [comment]
Sorry for having missed this. Maybe: `When none of the maintainers have commented nor self-assigned a PR in a delay of five working days, any maintainer of another component who feels comfortable reviewing the PR can assign it to themselves. To prevent misunderstandings, maintainers should not hesitate to announce in advance when they shall be unavailable for more than five working days.`?

**Comment by Zimmi48** [comment]
Thanks!

**Comment by SkySkimmer** [comment]
Who is in charge of merging this?

**Comment by ejgallego** [comment]
> Who is in charge of merging this?

Maybe @vbgl could drive this?

**Comment by maximedenes** [comment]
I can. But maybe @Zimmi48 wants to rebase to trigger his codeowners trick.

**Comment by Zimmi48** [comment]
Done.

**Review by SkySkimmer — APPROVED** [review]

**Comment by Zimmi48** [comment]
Since this PR hadn't yet been merged, I updated it to also mention the practice of code owner teams.

**Comment by SkySkimmer** [comment]
Are we waiting for something to merge this?

**Comment by maximedenes** [comment]
No, sorry, will merge now.

**Pull request merged**
